test/parallel/test-vm-debug-context.js

```diff
@@ -53,6 +53,17 @@ assert.strictEqual(vm.runInDebugContext(undefined), undefined);
   assert.equal(breaks, 1);
 })();
 
+// Can set listeners and breakpoints on a single line file
+(function() {
+  const Debug = vm.runInDebugContext('Debug');
+  const fn = require(common.fixturesDir + '/exports-function-with-param');
+
+  Debug.setListener(common.mustCall(function(e, s, d) {}));
```

Comment:
Is the `common.mustCall(...)` reliable here?  In my experience the listener frequently gets called for more than just breakpoint events.  I'd check manually that it gets called at least one with `d.constructor.name === 'BreakEvent'`.  Maybe give it a more descriptive name than just `d`. :-)